api-reference/lease/find-tenants.mdx

```diff
@@ -138,7 +138,17 @@ Status Code - 400
 ```
 
 Status Code - 401
-```json Invalid/Expired Authorization header bearer token value
+```json Expired Authorization header bearer token value
```

Comment:
this looks identical to the one in find-leases-for-tenant, are they able to be shared?

Reply:
I think sharing strings across markdown files is complicated. I don't think it's required here but if we prefer to do it this stackoverflow page describes a potential solution using pandoc and referencing multiple documents.

Stackoverflow page - [link](https://stackoverflow.com/questions/4779582/markdown-and-including-multiple-files)
Pandoc page - [link](https://pandoc.org/)

Reply:
Yes! I've added an issue for this previously https://github.com/Travtus/adam-api-docs/issues/8. I only found out you can define these shared snippets and re-use them late into writing these docs so I didn't do it initially, so I plan on doing this for all re-used code later in the future.

Reply:
For reference, this is the Mintlify functionality I've mentioned https://mintlify.com/docs/writing-content/reusable-snippets. Looks like they built something to make strings re-usable in mdx!